cypress/integration/fee-fine/fee-fine-history.spec.js

```diff
@@ -0,0 +1,139 @@
+import moment from 'moment';
+import permissions from '../../support/dictionary/permissions';
+import TestTypes from '../../support/dictionary/testTypes';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import Users from '../../support/fragments/users/users';
+import UsersOwners, { getNewOwner } from '../../support/fragments/settings/users/usersOwners';
+import TransferAccounts, { getNewTransferAccount } from '../../support/fragments/settings/users/transferAccounts';
+import ManualCharges from '../../support/fragments/settings/users/manualCharges';
+import PaymentMethods from '../../support/fragments/settings/users/paymentMethods';
+import WaiveReasons, { getNewWaiveReason } from '../../support/fragments/settings/users/waiveReasons';
+import RefundReasons, { getNewRefundReason } from '../../support/fragments/settings/users/refundReasons';
+import NewFeeFine from '../../support/fragments/users/newFeeFine';
+import AppPaths from '../../support/fragments/app-paths';
+import FeeFinesDetails from '../../support/fragments/users/feeFineDetails';
+import PayFeeFaine from '../../support/fragments/users/payFeeFaine';
+import AddNewStaffInfo from '../../support/fragments/users/addNewStaffInfo';
+import { HTML } from '../../../interactors';
+
+//The test fails because of the bug.
+describe('Fee/Fine history ', () => {
+  const userData = {
+    permissions: [
+      permissions.uiUsersView.gui,
+      permissions.uiUsersfeefinesCRUD.gui,
+      permissions.uiUsersViewServicePoints.gui,
+      permissions.uiUsersfeefinesView.gui,
+      permissions.uiUsersManualCharge.gui,
+      permissions.uiUsersManualPay.gui,
+      permissions.uiUserAccounts.gui
+    ],
+  };
+  const ownerData = {};
+  let feeFineType = {};
+  const paymentMethod = {};
+  const waiveReason = {};
+  const refundReason = {};
+  const transferAccount = getNewTransferAccount();
+  let feeFineAccount;
+  let servicePointId;
+  const newStaffInfoMessage = 'information to check';
+
+  before('Create owner, transfer account, feeFineType, paymentMethod, waiveReason, refundReason, user, feeFine', () => {
+    cy.getAdminToken();
+    ServicePoints.getViaApi({ limit: 1 }).then((servicePoints) => { servicePointId = servicePoints[0].id; });
+
+    UsersOwners.createViaApi(getNewOwner('owner')).then(({ id, ownerName }) => {
+      ownerData.name = ownerName;
+      ownerData.id = id;
+    })
+      .then(() => {
+        TransferAccounts.createViaApi({ ...transferAccount, ownerId: ownerData.id }); // is not created
+        ManualCharges.createViaApi({ ...ManualCharges.defaultFeeFineType, ownerId: ownerData.id }).then(manualCharge => {
+          feeFineType = { id:  manualCharge.id, name: manualCharge.feeFineType, amount: manualCharge.amount };
+        });
+        PaymentMethods.createViaApi(ownerData.id).then(({ name, id }) => {
```

Comment:
can't see clearing of created payment method

Reply:
I create the delete method for it